_posts/2016-02-24-session-user-flow.markdown

```diff
@@ -0,0 +1,64 @@
+---
+layout: post
+title: "Use session variables to optimize your user flow"
+date: 2016-02-24 13:37:00
+categories: ["ruby", "rails", "sessions"]
+author: "schmierkov"
+---
+
+Sessions provide you a nice little data storage feature where the application does not need to get the informations directly from the database. So you do not have to persist data in your database and can easily store info to a user on the fly. This is a nice way to enhance the user experience on your page.
```

Comment:
@schmierkov "get the informations" should be "get the information" 
"easily store info to a user on the fly" should be "easily store info about the user on the fly"